exercises/book-store/canonical-data.json

```diff
@@ -0,0 +1,76 @@
+{
+   "#": [
+     "Not sure what to type here, come back to this later."
+   ],
+   "cases": [
+      {
+         "description": "TestSingleBookPrice",
+         "expected": 8.0
+      },
+      {
+         "description": "TestBasketContainsMyBooks",
```

Comment:
May be too simple of a test.  It is simply echoing back the contents of its basket.  It should equal what was placed in the basket.